src/background/browser-permissions-tracker.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { Interpreter } from 'background/interpreter';
+import { BrowserAdapter } from 'common/browser-adapters/browser-adapter';
+import { Logger } from 'common/logging/logger';
+import { Message } from 'common/message';
+import { Messages } from 'common/messages';
+
+export const allUrlAndFilePermissions: string = '*://*/*';
```

Comment:
Instead of storing this, you could just store the permissions object that we always ask for.